tikv-client/src/main/java/com/pingcap/tikv/meta/TiDAGRequest.java

```diff
@@ -528,6 +540,7 @@ private void pushDownOrderBy(
     addPushDownOrderBys();
   }
 
+  @SuppressWarnings("SuspiciousMethodCalls")
```

Comment:
not necessary. You should change the code instead if you get SuspiciousMethodCalls warning.